po/en.po

```diff
@@ -622,17 +756,24 @@ msgid ""
 "`requests`와 `limits`를 둘 다 적어주셔야 합니다. Quota에 대한 내용은 [이 문"
 "서](./quota.md)를 확인해주시길 바랍니다."
 msgstr ""
+"Please note that by default, a Resource Quota is applied to all users. 
+Therefore, you need to specify the resources you request accurately. 
+Since both Request and Limit are restricted by the Quota, you should either write only `limits` as in the above example or specify both `requests` and `limits`. 
```

Comment:
```suggestion
Since both request and limit are restricted by the quota, you should either write only `limits` as in the above example or specify both `requests` and `limits`. 
```